src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.person;
+
+public class Block {
```

Comment:
You are expected to provide some javadoc style comments for classes and methods.